tests/e2e/cucumber/features/smoke/share.oc10.feature

```diff
@@ -44,9 +44,9 @@ Feature: share
     And "Brian" copies the following resource using dropdown-menu
       | resource                | to       |
       | Shares/folder_to_shared | Personal |
-    When "Brian" deletes the following resources
-      | resource                                    |
-      | Shares/folder_to_customShared/lorem-big.txt |
+    When "Brian" deletes the following resources using the sidebar panel
+      | resource      | from                          |
+      | lorem-big.txt | Shares/folder_to_customShared |
```

Comment:
This was better. :thinking: 
```suggestion
    When "Brian" deletes the following resources using the sidebar panel
      | resource                                    |
      | Shares/folder_to_customShared/lorem-big.txt |
```

Reply:
accumulating the path to folder may be good option if we need to upload more `files` on the same path rather than looping it various times.